charts/cadc-tap/templates/tap-deployment.yaml

```
@@ -100,6 +100,13 @@ spec:
                 -Dbase_url={{ .Values.global.baseUrl }}
                 -Dca.nrc.cadc.util.PropertiesReader.dir=/etc/creds/
                 -Xmx{{ .Values.config.jvmMaxHeapSize }}
+            {{- if and (eq .Values.config.backend "qserv") (eq .Values.config.qserv.passwordEnabled true) }}
```

Nit: rather than `(eq ... true)` you can just use the value itself for its truth value.
```suggestion
            {{- if (and (eq .Values.config.backend "qserv") .Values.config.qserv.passwordEnabled) }}
```
Pretty sure that works.